paasta_tools/api/views/instance.py

```diff
@@ -138,3 +139,20 @@ def instance_status(request):
         raise InstanceFailure(error_message, 500)
 
     return instance_status
+
+
+@view_config(route_name='service.instance.tasks', request_method='GET', renderer='json')
+def instance_tasks(request):
+    status = instance_status(request)
+    task_id = request.swagger_data.get('task_id', None)
+    slave_hostname = request.swagger_data.get('slave_hostname', None)
+    try:
+        mstatus = status['marathon']
+    except KeyError:
+        raise InstanceFailure("Only marathon tasks supported", 400)
+    tasks = get_tasks_from_app_id(mstatus['app_id'], slave_hostname=slave_hostname, task_id=task_id)
+    tasks = [{'task_id': task['id'],
```

Comment:
sorry to be a pain, but I'd prefer:

```
/service/{service}/instance/{instance}/tasks
['task_1','task_2','task_3']

/service/{service}/instance/{instance}/tasks/{task_id}
{
   'slave_id': ...
   'slave_hostname': ....
   'container_id': ...
}
```

also, why deliver such a subset of information? Isn't there a bunch more info available from the respective marathon/mesos apis?